docs/dev/guide/custom-aa-tutorial.md

```diff
@@ -12,15 +12,21 @@ It is also assumed that you already have some experience working on Ethereum.
 
 ## Installing dependencies
 
-We will use the zkSync hardhat plugin for developing this contract. Firstly, we should install all the dependencies for it: 
+We will use the zkSync hardhat plugin for developing this contract. Firstly, we should install all the dependencies for it:
 
 ```
-mkdir cross-chain-tutorial
-cd cross-chain-tutorial
+mkdir custom-aa-tutorial
+cd custom-aa-tutorial
 yarn init -y
 yarn add -D typescript ts-node ethers zksync-web3 hardhat @matterlabs/hardhat-zksync-solc @matterlabs/hardhat-zksync-deploy
 ```
 
+##
```

Comment:
empty headers?

Reply:
yes
It looks  like a separate  paragraph, but it is without a header, since it is small.Looks good IMHO 
<img width="922" alt="image" src="https://user-images.githubusercontent.com/72273339/176871806-c5276d9f-870d-4092-a7c0-2284197f4936.png">

Reply:
`##` is a header. Using it in this way is a hack. 

You could just do it like "Seconly, create the ...". No need to use the `##` here

Reply:
done